core/src/main/java/com/linecorp/armeria/client/TlsProviderBuilder.java

```diff
@@ -0,0 +1,19 @@
+package com.linecorp.armeria.client;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TlsProviderBuilder {
```

Comment:
Should we move this and related classes to `common` so that both clients and servers use them.